### ISSUE TITLE
Working POS mobile payment solution

### ISSUE THREAD

**Issue by pjv** [kilbot/WooCommerce-POS#210]
Hi,

Sorry to post this as an issue, but couldn't figure out any other way to reach either support for WooCommerce-POS (I emailed but received a canned response), or anyone else who is currently using the plugin in production.

We have an online WooCommerce shop (in the USA) and we need to add a mobile POS solution to sell at a convention. We use Stripe for CC processing. Our staff will have both Android phones and iPhones at the event. WooCommerce POS Pro looks like it might be a good solution for us, but browsing through support threads and github issues, it’s not clear whether there is currently production support for mobile card readers (attached to Android phones and/or iPhones). 

We are anticipating too many sales to allow us to be punching in CC numbers manually and my experience with photo-scanning cards with phone apps is that it is too hit and miss.

Can anyone tell me if there are any currently working hardware card scanning solutions that are compatible with WooCommerce POS Pro with the Stripe payment gateway?

**Comment by glaing** [comment]
Hi,

I used Square (https://squareup.com/) for this when we ran some pop-up shops last year.

We used the prebuilt ‘Credit Card’ WooCommerce POS payment gateway, so we had to manually type the transaction amount in to the phone before we took the payment. It wasn’t ideal, but it got the job done.

I was thinking about writing a plugin to connect the two together at some point, so it would write the Square transaction id back to WooCommerce (as it would make reconciling the transactions easier in my accounting software) and it would pass the order number and amount over to Square, to take away the possibility of mis-typing the amount.

If you do find an ‘in-person’ solution for Stripe, I would be interested to find out more.

I hope you manage to work something out!
Cheers,
George

**Comment by pjv** [comment]
Much appreciated @glaing.

As far as I can tell from 3 month old disqus comments [here](https://docs.wcpos.com/en/checkout/payment-gateways.html), WooCommerce POS Pro is not compatible with the Stripe payment gateway and @kilbot seems to be too busy working on a new version of the client to provide support for the existing version.

If that's true, I'll need to find another solution for our upcoming show. Too bad because WooCommerce POS would fit our use-case perfectly if it worked.

**Comment by glaing** [comment]
Yes, but I would imagine that the Stripe Gateway not being supported is more due to the fact that Stripe doesn't have official hardware (although it does seem they are working on this.. https://stripe.com/terminal) and that the current WooCommerce Stripe Gateway Plugin is for online use, so if it is used to take an in-person payment with the customer having to provide the 3 digit security code and not being able to type their PIN number in, technically this would be in breach of PCI DSS rules. (My knowledge is based rules for Chip and PIN Cards in UK stores, but I would guess that it would be the same in the USA).

This is why I went down the Square route (iZettle and PayPal both also have hardware options, which could work equally well).
